From 3bf11cee3eceacb7f3ec177fb676ba36f373d369 Mon Sep 17 00:00:00 2001
From: Christian Boltz <apparmor@cboltz.de>
Date: Sun, 14 Oct 2018 18:10:46 +0200
Subject: [PATCH] Fix syntax error in rc.apparmor.functions

This bug was introduced in
- https://gitlab.com/apparmor/apparmor/merge_requests/230
- commit c974dd0d071149213892f03de0869b860705bda0 (master)
- commit 9987a7ec9c0fc7fd9ac6831152dc77fb477fd04a (2.13 branch)
---
 parser/rc.apparmor.functions | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/parser/rc.apparmor.functions b/parser/rc.apparmor.functions
index 516edf11..9edded52 100644
--- a/rc.apparmor.functions
+++ b/rc.apparmor.functions
@@ -128,7 +128,7 @@ skip_profile() {
 		return 2
 	fi
 	if echo "${profile}" | egrep -q '^.+\.new-[0-9\.]+_[0-9]+$'; then
-		return 2 ;;
+		return 2
 	fi
 
 	return 0
-- 
2.18.1

